vendor/symfony/polyfill-mbstring/bootstrap.php

```diff
@@ -0,0 +1,147 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+use Symfony\Polyfill\Mbstring as p;
+
+if (\PHP_VERSION_ID >= 80000) {
+    return require __DIR__.'/bootstrap80.php';
+}
+
+if (!function_exists('mb_convert_encoding')) {
```

Comment:
not now, but it would be nice to change this logic to a switch case or something friendlier since it increases the complexity when doing it with ifs

Reply:
we can even handle it with ENUMS